nixos/modules/config/swap.nix

```diff
@@ -114,6 +114,18 @@ let
         '';
       };
 
+      discardPolicy = mkOption {
+        default = null;
+        example = "once";
+        type = types.nullOr (types.enum ["once" "pages" "both" ]);
+        description = ''
+        Specify the discard policy for the swap device. If "once", then the whole swap space is
+        discarded at swapon invocation. If "pages", asynchronous discard on freed pages is performed,
+        before returning to the available pages pool. With "both", both policies are activated.  See
+        swapon(8) for more information.
```

Comment:
```suggestion
          Specify the discard policy for the swap device. If "once", then the
          whole swap space is discarded at swapon invocation. If "pages",
          asynchronous discard on freed pages is performed, before returning to
          the available pages pool. With "both", both policies are activated.
          See swapon(8) for more information.
```
indentation

Reply:
For future reference: line break at around 80 for human-readable text is the recommended style?

Reply:
The main problem was the lack of indentation, but yes I also wrapped it to 80 characters.
There's no real rule:
https://logs.nix.samueldr.com/nixos-dev/2021-05-18#1621331047-1621331140;

Reply:
Please add two spaces for indentation like in the suggestion and others multiline blocks https://github.com/NixOS/nixpkgs/blob/4bda1320a4e066e365482619ba649977b35c9fc4/nixos/modules/config/swap.nix#L17-L27.

Reply:
oops sorry, totally missed the spaces